c++/src/capnp/persistent.capnp.c++

```diff
@@ -69,8 +69,8 @@ static const ::capnp::_::RawSchema* const d_c8cb212fcd9f5691[] = {
 };
 static const uint16_t m_c8cb212fcd9f5691[] = {0};
 const ::capnp::_::RawBrandedSchema::Dependency bd_c8cb212fcd9f5691[] = {
-  { 33554432,  ::capnp::Persistent< ::capnp::AnyPointer,  ::capnp::AnyPointer>::SaveParams::_capnpPrivate::brand },
-  { 50331648,  ::capnp::Persistent< ::capnp::AnyPointer,  ::capnp::AnyPointer>::SaveResults::_capnpPrivate::brand },
+  { 33554432,  ::capnp::Persistent< ::capnp::AnyPointer,  ::capnp::AnyPointer>::SaveParams::_capnpPrivate::brand() },
```

Comment:
So do these initializers run code at startup when compiled with MSVC?

If so that's pretty unfortunate -- dynamic startup-time initializers are something that the Chrome team has in the past worked vigorously to eliminate, to improve startup time. The main problem is not the execution time of the code itself but the fact that the code must be paged in from disk in order to execute.

But maybe MSVC is actually able to optimize these away? Can you check somehow if any dynamic initialization code is being generated?

If dynamic initializers are being generated, maybe we could avoid them by manual inlining? That is, instead of a call to `brand()`, actually refer to `&schema->defaultBrand` directly here?